test/evp_test.c

```diff
@@ -2577,6 +2577,7 @@ typedef struct pkey_data_st {
     size_t output_len;
     STACK_OF(OPENSSL_STRING) *init_controls; /* collection of controls */
     STACK_OF(OPENSSL_STRING) *controls; /* collection of controls */
+    STACK_OF(OPENSSL_STRING) *mu_controls; /* collection of controls */
```

Comment:
Is another specific list required?

`evp_test` tries to be algorithm agnostic.  At least a little.

Reply:
You understand that this is doing an external mu as a digest followed by passing this into a existing sign or verify.
So I cant reuse the controls, The only option would be to dup quite a bit of code which I thought was worse..

Reply:
Would it not be possible to reused `controls` for the purpose??